package.json

```diff
@@ -15,7 +15,7 @@
     "db:migrate": "postgrator",
     "db:down": "docker compose down",
     "lint": "eslint --ext .ts,.js ./src",
-    "test": "tap && c8 report --check-coverage --temp-directory .tap/coverage --100"
+    "test": "c8 --check-coverage --100 node --test"
```

Comment:
shall we consider running this as a workspace command instead, so each workspace runs its own tests? e.g. `npm test --workspaces`

Reply:
if there aren't any obvious benefits, no, we can keep it as it is. let's just say that conceptually it would be more correct that each package in the workspace runs its own tests, but I think it's an acceptable compromise to do it this way